modules/server/.env.schema

```
@@ -14,4 +14,5 @@ MAX_LIVE_VERSIONS=3
 MAX_RESULTS_WINDOW=10000
 PING_MS=2200
 PING_PATH=/ping
-PORT=5050
\ No newline at end of file
+PORT=5050
+ENABLE_AGGREGATION_MODE=false
```

lol "for consistency" 
(i.e. not sure why it was missing to begin with 🤔)
```suggestion
ENABLE_AGGREGATION_MODE=false

```